export_assets_to_android_ios.jsx

```diff
@@ -111,12 +147,63 @@ function exportToFile(scaleFactor, resIdentifier, os) {
 	}
 };
 
+function createSourceSelector(name, array, parent) {
+    var panel = parent.add("panel", undefined, name);
+    var dropDown = panel.add("DropDownList", undefined, "source");
+    for (var i = 0; i < array.length; i++) {
+        var select = dropDown.add("item", array[i].name);
+        select.item = array[i];
+        if(array[i].scaleFactor == options.source)
+            select.selected = true;
+    }
+    dropDown.onChange = function () {
+        options.source = this.selection.item.scaleFactor;
+    };
+}
+
+function createSizePanel(name, array, parent) {
+    var panel = parent.add("panel", undefined, name);
+    for(var i = 0; i < array.length; i++) {
+        var cb = panel.add("checkbox", undefined, "\u00A0" + array[i]);
+        cb.item = array[i];
+        if(selectedSizes.indexOf(cb.item) > -1)
+            cb.value = true;
+        cb.onclick = function () {
+            if(this.value) {
+                selectedSizes.push(this.item);
+            } else {
+                selectedSizes.splice(selectedSizes.indexOf(this.item), 1);
+            }
+        }
+    }
+}
+
+function createSizeSourcePanel(name, array, parent)
+{
```

Comment:
`{` braces should be on the same line. Please use the existing style for any new code.